src/Cpp/1-getting-started/1-HelloWindow/Input/Input.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include "Keyboard.hpp"
+#include "Mouse.hpp"
+
+#include <cstdint>
+#include <memory>
+
+// ReSharper disable once CppInconsistentNaming
```

Comment:
Comment seems out of place?

Reply:
As mentioned earlier was about `GLFWwindow`  which according to ReSharper should be `GlfwWindow` (will be addressed by clang-lint at some point)